src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -106,6 +130,22 @@ export const DefaultPopulationSelectorDefinitions = [
   },
 ];
 
+export const MeasureImprovmentNotation = [
```

Comment:
Typo _MeasureImprovementNotation_

Reply:
I copied syntax for the DefaultPopulationSelectorDefinitions above. Uncertain if export is needed

Reply:
aah ok, no problem. I just wondered if there is any other instance that is using it.